alarm_base/src/lend.rs

```diff
@@ -12,22 +12,20 @@
 //! or, "So You've Always Wished `*mut u8` Could `impl Drop`..."
 use alloc::allocator::{Alloc, AllocErr, Layout};
 use core::{mem, ops, ptr};
-
 use spin;
 
 /// An allocator that can provide borrowed handles.
 pub trait Lend {
-
     /// The type of the allocator providing the allocation.
     ///
     /// This has to be an associated type rather than `Self`
     /// so that `Lend` can be implemented for e.g. `Mutex<A>`.
     type Allocator: Alloc;
 
     /// Borrow an allocation for a `T` from this lender.
-    fn borrow<'a, T>(&'a self)
-                    -> Result<Borrowed<'a, T, Self::Allocator>, AllocErr>;
-
+    fn borrow<'a, T>(
+        &'a self,
+    ) -> Result<Borrowed<'a, T, Self::Allocator>, AllocErr>;
```

Comment:
Ugh, I hate the way this looks; it would be really nice if `rustfmt` would still allow us to set the "Visual" indent style _just_ for function declarations, like we used to be able to. I'm considering just setting the visual indent style, but that can lead to things like closures being indented weirdly.

Reply:
Hahah, the new rustfmt goes with block style for almost everything. I used to have an opinion like that about the positioning of the 'where' keyword in the new rfc style, but honestly you just get used to it.